Openvstorage/Framework documentation/readme.md

```diff
@@ -0,0 +1,34 @@
+## Introduction
+
+### Structure of the framework and its repos
+
+#### Framework
+This is the core of the ovs  [framework](todo insert link). Contains all logic for
+ - CLI and the GUI work
+ - plugin loading
+ - service management
+ - configuration management
+ - DAL stuff
+ 
+
+ 
+ #### Framework-extensions
+Framework extensions contain logic that  
+ - is shared across multiple repos such as the framework, asd-manager or iscsi-manager,
+ and should be installed regardless of which one of these is installed.
+ - function as an interface and should be implemented by one of these managers
+ 
+ #### Alba-asd and iSCSI manager
+ These managers contain logic to function as a standalone nodes
```

Comment:
Microservices that are only responsible for a single thing. These services can run on different nodes, regardless of where the Framework cluster lives.
They are accessed through their HTTP API after installing the corresponding Framework plugin